pkg/streamingpromql/operators/functions/range_vectors_test.go

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package functions
+
+import (
+	"math"
+	"testing"
+
+	"github.com/prometheus/prometheus/promql"
+
+	"github.com/grafana/mimir/pkg/streamingpromql/limiting"
+	"github.com/grafana/mimir/pkg/streamingpromql/types"
+)
+
+func TestDoubleExponentialSmoothing_Validation(t *testing.T) {
```

Comment:
There's no need to test this with a Go test: we can make assertions about errors in `.test` files with `expected_fail_msg`.

Reply:
Removed the go test and add expted_fail_message promql.test fe571cbc0242191f5b4db79dfee7af3c69f8a1be